.github/workflows/unit-tests.yml

```diff
@@ -27,6 +27,50 @@ env:
   ADDITIONAL_CMAKE_FLAGS: -DRTTR_ENABLE_BENCHMARKS=ON
 
 jobs:
+  Windows:
+    strategy:
+      matrix:
+        include:
+          - { os: windows-2019, generator: Visual Studio 16 2019, type: Debug,   platform: Win32}
```

Comment:
I honestly would skip 2019

Reply:
I'd rather skip the Win32 builds, who still has 32 bit Windows? I found it useful to have at least 2 different compilers to shake out compiler issues we might run into. I.e. if it works for 2019 but fails in 2022 that would be a good information.

I'll merge this now and we can remove some later if we still want to.